Update Sanbox building instruction

YuWei-CH opened the pull request:
1. button here
2. click and jump to a new window
3. add border to input box
4. Fix Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers

YuWei-CH committed:
Update with Builder Sandbox Instruction

1. make "here" link as button and click to open a new tab.
2.Add border to input box

YuWei-CH committed:
Fix security risk Error

Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers

tanerdurmaz reviewed (changes requested):
need to change buttons details in discord

YuWei-CH committed:
Merge branch 'OSFD-Winter:main' into main

YuWei-CH committed:
Update sandbox.tsx

1. make all buttons without border and white_bgcolor
2. button the first "Here" and make it jump to new page after click
3. modify the layout of "Choose File" and "Upload" button
4. Add "Cancel" and "Create NFT Series" buttons.

Next to do:  Complete the functions of the above two buttons

YuWei-CH committed:
Fix error in CodeFactor Check

Fix:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.

YuWei-CH commented:
> need to change buttons details in discord

Changed!
